initcwnd.rb

```diff
@@ -47,7 +47,7 @@
 File.delete("initcwnd.pcap") if File.exist?("initcwnd.pcap")
 
 pid = fork do
-  exec("tcpdump", "-i", interface, "-w", "initcwnd.pcap", "host", uri.host, "and", "port", "443")
+  exec("tcpdump", "-i", interface, "-w", "initcwnd.pcap", "--packet-buffered", "host", uri.host, "and", "port", "443")
```

Comment:
Why did you need this? Does it work on MacOS?

Reply:
From the commit message:
> Use `--packet-buffered` so that tcpdump will write packets into the dump as it processes
  them, rather than waiting for a buffer to fill up.

This is a standard option of tcpdump, so while I didn't try I would assume it works on macOS as well.